reports/m1_proposal.md

```diff
@@ -0,0 +1,21 @@
+**Description of the data**
+
+The original raw data is downloaded from this URL <https://thepulseofthenation.com/#future>. This dataset is the pull result from Card Against Humanity to the nationally representative of American public about popular social concerns during time period August 1st to 15th 2018. The survey was conducted via face to face, cell phone, and land lines, and respondents' demographic info and opinion choices on survey questions were collected.
+
+The raw dataset contains 800 x 24 datapoints, including collected answers from 800 respondents about 7 demographical questions and 14 opinion questions. It contains 11% of missing data points mainly from redundant 'other' columns.
+
+In order to better understand public opinion of different demographic groups, we have done some initial data preprocessing and data engineering to get prepared for visualization. For data preprocessing, we dropped 5 irrelevant, or vacant columns. We renamed column title into brief phrase form. We added the 'political_spectrum_score' column based on numerical conversion of the 'ideaology' choice to help understand political spectrum. Further, we added the 'time_answered' column to reflect this survey period for dashboard sustainability purpose if there are more incoming surveies in future month.
+
+The cleaned dataset contains 800 rows and 21 columns. It is structured with a focus on public opinions and demographic data, tailored to understand societal trends and political leanings. Key characteristics of this dataset include:
```

Comment:
We have 21 columns in final. but based on previous description, we had 21 original columns 7 demographic + 14 survey questions = 21. And we removed 5 and added 2 so 21 -5 +2 = 18. Can we double check this

Reply:
The original is 24 (*8 demographic question are in *10 columns  (two nearly empty columns called other) + 14 survey questions) 
We removed five and added two, finalized version is 21 columns.